From 6ac174103716ea5ab645eb8e70bed4fe50e291e2 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sat, 29 Jun 2019 16:10:37 -0500
Subject: [PATCH 2/8] use the correct attributes for ___errno_location

In the public header, __errno_location is declared with the "const"
attribute, conditional on __GNUC__. Ensure that its internal alias has
the same attributes.
---
 src/include/errno.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/include/errno.h b/src/include/errno.h
index 54a38ff4..8ec49377 100644
--- a/src/include/errno.h
+++ b/src/include/errno.h
@@ -3,6 +3,9 @@
 
 #include "../../include/errno.h"
 
+#ifdef __GNUC__
+__attribute__((const))
+#endif
 hidden int *___errno_location(void);
 
 #undef errno
-- 
2.21.0

